app/managers.py

```diff
@@ -0,0 +1,48 @@
+import sqlite3
+
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "Actors"
+
+    def all(self) -> None:
+        actor_cursor = self._connection.execute(
+            f"SELECT id, first_name, last_name FROM {self.table_name}"
```

Comment:
use * to select all columns from a table

Reply:
Done)